x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_custom_source/save_custom.tsx

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiSpacer,
+  EuiText,
+  EuiTextAlign,
+  EuiTitle,
+  EuiLink,
+  EuiPanel,
+  EuiHorizontalRule,
+  EuiCallOut,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { EuiButtonTo, EuiLinkTo } from '../../../../../../shared/react_router_helpers';
+import { AppLogic } from '../../../../../app_logic';
+import { API_KEY_LABEL } from '../../../../../constants';
+import { SOURCES_PATH, getSourcesPath, API_KEYS_PATH } from '../../../../../routes';
+
+import { SourceIdentifier } from '../../source_identifier';
+
+import { AddSourceHeader } from '../add_source_header';
+import { SAVE_CUSTOM_BODY1 as READY_TO_ACCEPT_REQUESTS_LABEL } from '../constants';
+
+import { AddCustomSourceLogic } from './add_custom_source_logic';
+
+export const SaveCustom: React.FC = () => {
+  const { newCustomSource, sourceData } = useValues(AddCustomSourceLogic);
+  const { isOrganization } = useValues(AppLogic);
+  const {
+    serviceType,
+    configuration: { githubRepository, documentationUrl },
+    name,
+    categories = [],
+  } = sourceData;
+
+  return (
+    <>
+      <AddSourceHeader name={name} serviceType={serviceType} categories={categories} />
+      <EuiSpacer />
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiPanel paddingSize="l" hasShadow={false} color="subdued">
+            <EuiFlexGroup
+              direction="column"
+              alignItems="center"
+              style={{ marginTop: 'auto', marginBottom: 'auto' }}
+            >
+              <EuiFlexItem>
+                <EuiIcon type="checkInCircleFilled" color="#42CC89" size="xxl" />
```

Comment:
You'll want to check this in dark mode, as hard-coded color values are ignored when themes change. The best thing in this case would be to reference a color variable.